Modules/Phasing.lua

```diff
@@ -320,6 +320,10 @@ local phases = {
     SUNA_AT_CAMP_OSUL = 1248,
     BAN_AT_OUTPOST = 1249,
     BAN_AT_CAMP_OSUL = 1250,
+    BEFORE_MANTID_INVASION = 1251,
+    AFTER_MANTID_INVASION = 1252,
```

Comment:
This phasing ID is not handled in the code below, so there is logic missing to it.